packages/react-tabs/src/components/Tab/useTabAnimatedIndicator.ts

```diff
@@ -37,47 +31,42 @@ const useActiveIndicatorStyles = makeStyles({
       },
     },
   },
-  animatedVertical: {
+  horizontal: {
     ':after': {
-      transitionProperty: 'top, height, bottom',
-      transitionDuration:
-        `${pendingAnimationDurationTokens.slow},` +
-        `${pendingAnimationDurationTokens.slow},` +
-        `${pendingAnimationDurationTokens.slow}`,
-      transitionTimingFunction:
-        `${pendingAnimationEasingTokens.declerateMax},` +
-        `${pendingAnimationEasingTokens.declerateMax},` +
-        `${pendingAnimationEasingTokens.declerateMax}`,
+      transformOrigin: 'left',
+      transform: `translateX(var(${tabIndicatorCssVars_unstable.offsetVar}))
+    scaleX(var(${tabIndicatorCssVars_unstable.scaleVar}))`,
     },
-    '@media (prefers-reduced-motion: reduce)': {
-      ':after': {
-        transitionProperty: 'none',
-        transitionDuration: '0ms',
-      },
+  },
+  vertical: {
+    ':after': {
+      transformOrigin: 'top',
+      transform: `translateY(var(${tabIndicatorCssVars_unstable.offsetVar}))
+        scaleY(var(${tabIndicatorCssVars_unstable.scaleVar}))`,
     },
   },
   mediumHorizontal: {
     ':after': {
-      left: `calc(var(${tabIndicatorCssVars_unstable.startOffsetVar}) + ${tabIndicatorPadding.mediumHorizontal})`,
-      right: `calc(var(${tabIndicatorCssVars_unstable.endOffsetVar}) + ${tabIndicatorPadding.mediumHorizontal})`,
+      left: tabIndicatorPadding.mediumHorizontal,
+      right: tabIndicatorPadding.mediumHorizontal,
```

Comment:
I think you can actually take all of these out entirely now, since they're already styled in `useTabStyles`